Makefile

```diff
@@ -1,10 +1,10 @@
 PRG=Gwion
 LDFLAGS = -lsoundpipe
-LDFLAGS += -g -lm -pthread -lasound -ljack -lsndfile -fno-strict-aliasing
-LDFLAGS += -std=c99 -O3 -mfpmath=sse -mtune=core2 -freg-struct-return -ldl -rdynamic -lrt -lsoundio
+LDFLAGS += -lm -pthread -lasound -ljack -lsndfile 
+LDFLAGS += -ldl -rdynamic -lrt -lsoundio
 
-CFLAGS+=-Iinclude
-CC = cc
+CFLAGS += -Iinclude -g -std=c99 -O3 -mfpmath=sse -mtune=core2 -freg-struct-return 
+CFLAGS += -fno-strict-aliasing -Wall
```

Comment:
I've added -Wall here to your CFLAGS. Optional, but I really like it as it exposes all the wounds and possible points of trouble that would otherwise go unnoticed.